### PR TITLE
SEO-173276-WPF-Images-Redirection

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
Please find changes made under image section.
Regards,
Scholar.

### CODE REVIEW

File: wpf/Themes/Skin-Manager.md [hunk 1]
```diff
@@ -1194,8 +1194,8 @@ We have provided an example that demonstrates switching between the Window11Ligh
 
 Light Theme:
 
-![Customizing theme while run time in Light](Skin-Manager_images/Switching-theme-at-runtime-light1.png)
+![Customizing theme while run time in Light](skin-manager_images/wpf-themes-switching-theme-at-runtime-light.png)
```
Comment (MallikaRK) [inline wpf/Themes/Skin-Manager.md:1197]: change the image name as
wpf-skin-manager-light-theme

File: wpf/Themes/Skin-Manager.md [hunk 2]
```diff
@@ -1194,8 +1194,8 @@ We have provided an example that demonstrates switching between the Window11Ligh
...
 
 Dark Theme:
-![Customizing theme while run time in Dark](Skin-Manager_images/Switching-theme-at-runtime-Dark1.png)
+![Customizing theme while run time in Dark](skin-manager_images/wpf-themes-switching-theme-at-runtime-Dark.png)
```
Comment (MallikaRK) [inline wpf/Themes/Skin-Manager.md:1200]: Change the name as per previous comment

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Themes/Skin-Manager.md (hunk 1 above)
Comment (Scholar-jaruha) [reply wpf/Themes/Skin-Manager.md:1197]: changed

File: wpf/Themes/Skin-Manager.md (hunk 2 above)
Comment (Scholar-jaruha) [reply wpf/Themes/Skin-Manager.md:1200]: changed

File: wpf/Tabbed-MDI-Form/Getting-Started.md [hunk 3]
```diff
@@ -194,8 +194,6 @@ You can minimize the `MDI` window by setting the [CanMDIMinimize](https://help.s
 {% endhighlight %}
 {% endtabs %}
 
-![Minimizing MDI window in Document Container](Minimizing-MDI-window_images/Minimizing-MDI-window_img1.jpeg)
```
Comment (MallikaRK) [inline wpf/Tabbed-MDI-Form/Getting-Started.md:197]: Why you removed the image here?

File: wpf/Themes/Skin-Manager.md [hunk 4]
```diff
@@ -384,7 +384,7 @@ There are several ways for including the Syncfusion [SfSkinManager](https://help
 1) Add a reference to the `Syncfusion.SfSkinManager.WPF` assembly or [Syncfusion.SfSkinManager.WPF nuget package](https://www.nuget.org/packages/Syncfusion.SfSkinManager.WPF/) to the project.
 2) Import Syncfusion WPF schema `http://schemas.syncfusion.com/wpf` or the assembly namespace `Syncfusion.SfSkinManager` into a XAML page.
 
-![Add SfSkinManager Reference](Skin-Manager_images/WPF-SkinManager-Reference.png)
+![Add SfSkinManager Reference](skin-manager_images/wpf-theme-reference.png)
```
Comment (MallikaRK) [inline wpf/Themes/Skin-Manager.md:387]: Add platform name here. 
Add WPF SkinManager Reference - use this word in alt text. 
Check and update in all places.

File: wpf/Themes/Theme-Studio.md [hunk 5]
```diff
@@ -183,7 +183,7 @@ The WPF Theme Studio comes with various set of palettes for each theme variant.
 </tr>
 </table>
 
-![WPF Theme Studio Palette](ThemeStudio_images/ThemeStudio-Palette.gif)
+![WPF Theme Studio Palette](themestudio_images/wpf-themestudio-palette.gif)
```
Comment (MallikaRK) [inline wpf/Themes/Theme-Studio.md:186]: give hyphen in image name [all places]
theme-studio

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Tabbed-MDI-Form/Getting-Started.md (hunk 3 above)
Comment (Scholar-jaruha) [reply wpf/Tabbed-MDI-Form/Getting-Started.md:197]: I have reverted

File: wpf/Themes/Skin-Manager.md (hunk 4 above)
Comment (Scholar-jaruha) [reply wpf/Themes/Skin-Manager.md:387]: changed

File: wpf/Themes/Theme-Studio.md (hunk 5 above)
Comment (Scholar-jaruha) [reply wpf/Themes/Theme-Studio.md:186]: Added hyphen

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine

**Pull request merged**